docs/iris/src/userguide/real_and_lazy_data.rst

```diff
@@ -0,0 +1,225 @@
+.. _real_and_lazy_data:
+
+==================
+Real and Lazy Data
+==================
+
+What is Real and Lazy Data?
+---------------------------
+
+Every Iris cube contains an n-dimensional  data array, which could be real or
+lazy.
+
+Real data is contained in an array which has a shape, a data type, some other
+useful information and many data points, each of which use up a small
+allocation of memory.  This generally takes the form of a numpy array.
+
+Lazy data is contained in a conceptual array which retains the information
+about its real counterpart but has no actual data points, so its memory
+allocation is much smaller.  This will be in the form of a Dask array.
+
+Arrays in Iris can be converted flexibly(?) between their real and lazy states,
+although there are some limits to this process.  The advantage of using lazy
+data is that it has a small memory footprint, so certain operations
+(such as...?) can be much faster.  However, in order to perform other
```

Comment:
The advantage is quite right, but the application is not so accurate: lazy data allows you to load and manipulate datasets that you would otherwise not be able to fit into memory.

As it happens, it is likely that operations will be sped up, but this could be considered an indirect benefit:

* the calculations will not be performed immediately as you execute the lines of code that define the calculations. Instead they are **deferred** until you request a realised result (this happens when you call `compute` on a dask array)
* dask's parallel-processing schedulers allows the calculations to be performed in parallel when the realised result is requested, which can bring a significant performance improvement. However it is not _directly_ related to the smaller memory footprint.

I think both of these points are also worth adding here as they build the picture of the benefits of lazy data and its processing.